lib/card/content.rb

```diff
@@ -26,7 +26,7 @@ def card
     end
     
     def chunk_list
-      @opts[:chunk_list] || card.chunk_list
+      @opts[:chunk_list] || @format.chunk_list
```

Comment:
We do turn off chunk processing based on card types, and it might make sense that the kinds of chunks are more related to the card sets than the format classes.  On the other hand a format instance has a card and it's sets loaded for views.